repository_service_tuf/cli/admin/helpers.py

```diff
@@ -405,17 +377,24 @@ def _add_root_signatures_prompt(
 
         results = _filter_root_verification_results(root_result)
         keys = _print_keys_for_signing(results)
-
-        choice = _choose_signing_key_prompt(
-            len(keys), allow_skip=bool(root_result.signed)
-        )
-        if choice == -1:
+        key_choices = {
+            key.unrecognized_fields.get(KEY_NAME_FIELD, key.keyid): key
+            for key in keys
+        }
+        sign_options = list(key_choices.keys())
+        or_continue = ":"
+        if bool(root_result.signed):
```

Comment:
Interesting trick with the cast to bool, but I am not sure it's better than just checking if `len((root_result.signed) > 0`.
I think by using length you make it clearer what is the intention - check whether there is at least one key that  signed root.

Or will it probably be even clearer to check `len(root_md.signatures) > 1`?

Reply:
We only want to check if the initial signature is present.
I reused this code from the previous logic.